presto-mongodb/src/test/java/io/prestosql/plugin/mongodb/TestMongoDistributedQueriesLatest.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.mongodb;
+
+import com.google.common.collect.ImmutableMap;
+import io.prestosql.testing.QueryRunner;
+import io.prestosql.tpch.TpchTable;
+
+import static io.prestosql.plugin.mongodb.MongoQueryRunner.createMongoQueryRunner;
+
+public class TestMongoDistributedQueriesLatest
+        extends BaseMongoDistributedQueries
+{
+    @Override
+    protected QueryRunner createQueryRunner()
+            throws Exception
+    {
+        MongoServer server = closeAfterClass(new MongoServer("mongo:3.6.20"));
```

Comment:
Why latest is 3, and not 4?

Reply:
We are testing the latest 3.x here and there is a `TestMongo4DistributedQueriesLatest` for the latest in major version 4.